tests/test_all_rules.py

```diff
@@ -1040,6 +1041,30 @@ def test_event_dataset(self):
                     if validation_integrations_check and "event.dataset" in rule.contents.data.query:
                         raise validation_integrations_check
 
+    def test_min_stack_version_supported(self):
+        failures = []
+        # Load supported stack versions from stack-schema-map.yaml
+        stack_map_path = Path(f"{ETC_DIR}/stack-schema-map.yaml")
+        with Path.open(stack_map_path) as f:
+            stack_map = yaml.safe_load(f)
+
+        # Get the minimum supported stack version (as string)
+        min_supported = min([v for v in stack_map if not v.startswith("#") and isinstance(v, str)])
```

Comment:
If you follow Eric's suggestion, then you shouldn't have to parse and filter out #.